src/containers/SideNav/SideNav.js

```diff
@@ -81,11 +86,10 @@ function SideNav(props) {
 
   return (
     <CarbonSideNav
-      isFixedNav={expanded}
-      isRail={!expanded}
-      expanded={expanded}
-      isChildOfHeader={false}
       aria-label="Main navigation"
+      expanded
+      isChildOfHeader={false}
```

Comment:
Does this default to true in Carbon ?

Reply:
It does indeed https://github.com/carbon-design-system/carbon/blob/59cbf8d4a129babcc9fdd79ab6021928f7cbf7de/packages/react/src/components/UIShell/SideNav.js#L152